requirements/external.txt

```diff
@@ -0,0 +1 @@
+-e git://github.com/mozilla-iam/Flask-pyoidc.git@92a7b17e48c760534558e51e6d803fee12a684f8#egg=Flask-pyoidc
```

Comment:
@darkwing So the name "external.txt" threw me for a bit of a loop - I can reason about what a file named "vendor.txt" does in a project, but "external.txt" or "external" is a term I haven't encountered.  I thought the term and file name might have come from another programming language's community?

I'd prefer to change it to "vendor.txt", but I'm open to being persuaded otherwise. :smile: